.github/workflows/ci.yml

```
@@ -118,7 +118,7 @@ jobs:
             arch: "386"
           - os: windows-latest
             arch: amd64
-          - os: actuated-arm64-4cpu-4gb
+          - os: actuated-arm64-1cpu-1gb
```

I wouldn't suggest going this low, even the legacy hosted runners had more than this.

2cpu-8gb is probably the lowest you need to consider, thanks for being frugal and helping us to balance this shared resource.

---

I would follow @alexellis suggestion 

```suggestion
          - os: actuated-arm64-2cpu-8gb
```

---

We do not expect any customers to run with such low requirements and will not be able to support you if you chose to do so, it could result in some unexpected problem.

Please keep to 2cpu/8gb as a minimum. Thanks.